tests/theia-text-editor.test.ts

```diff
@@ -15,23 +15,41 @@
 // *****************************************************************************
 
 import { expect, test } from '@playwright/test';
-import { DefaultPreferences, PreferenceIds, TheiaApp, TheiaPreferenceView, TheiaTextEditor, TheiaWorkspace } from '@theia/playwright';
+import { DefaultPreferences, PreferenceIds, TheiaApp, TheiaAppLoader, TheiaExplorerView, TheiaPreferenceView, TheiaTextEditor, TheiaWorkspace } from '@theia/playwright';
 
 let textEditor: TheiaTextEditor;
+let app: TheiaApp;
 
-test.beforeAll(async ({ browser }) => {
-    const page = await browser.newPage();
+test.beforeAll(async ({ playwright, browser }) => {
     const ws = new TheiaWorkspace(['tests/resources/sample-files1']);
-    const app = await TheiaApp.loadApp(page, TheiaApp, ws);
+    app = await TheiaAppLoader.load({
+        playwright, useElectron: {
+            electronAppPath: 'theia/examples/electron'
+        }, browser
+    }, ws);
 
     // set auto-save preference to off
     const preferenceView = await app.openPreferences(TheiaPreferenceView);
     await preferenceView.setOptionsPreferenceById(PreferenceIds.Editor.AutoSave, DefaultPreferences.Editor.AutoSave.Off);
     await preferenceView.close();
 
+    // electron-specific workaround for multi root workspace
+    if (app.isElectron) {
+        const explorer = await app.openView(TheiaExplorerView);
+        await explorer.waitForVisibleFileNodes();
+        console.log(`app.workspace.path = ${app.workspace.path}`);
+        const workspaceBasename = app.workspace.path.split('/').pop();
+        if (workspaceBasename) {
+            await app.page.getByText(workspaceBasename).click();
+        }
+    }
```

Comment:
Should we maybe extract this into an own function that takes the `app` as a parameter to avoid repeating this code in several tests?

Reply:
Absolutely, I will adjust the PR!